src/main/java/software/amazon/smithy/lsp/SmithyTextDocumentService.java

```diff
@@ -142,12 +144,17 @@ public Optional<File> getRoot() {
     public void createProject(SmithyBuildExtensions ext, File root) {
         Either<Exception, SmithyProject> loaded = SmithyProject.load(ext, root);
         if (loaded.isRight()) {
-            this.project = loaded.getRight();
+            SmithyProject project = loaded.getRight();
```

Comment:
**thought:** this local is only here to avoid redundant warnings in IDEs saying that `this.project` is nullable in the lines below.